osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SnakingSliderBody.cs

```diff
@@ -125,7 +125,11 @@ private void updatePathSize()
         private void setRange(double p0, double p1)
         {
             if (p0 > p1)
-                MathHelper.Swap(ref p0, ref p1);
+            {
+                double temp = p0;
+                p0 = p1;
+                p1 = temp;
```

Comment:
Seeing that tuple deconstruction is in C# 7.0, I'd go with the python-derived
```csharp
(p0, p1) = (p1, p0);
```
No tuple allocation ([sharplab proof & comparison with the long-winded method](https://sharplab.io/#v2:C4LglgNgPgAgTARgLACgYGYAE9MGFMDeqmJ2WMALJgMoDuAhgA4AqAroxAKYA8zAfAAoATpwBmmZpgAeAGkwjxkgJ4BKYqQ20AFpxETMITAGdgQ1gGNg6kkRQb7pAbMyrMAXkwClcqSoDc1pgAvoGBGNhUdEwAsvQAdqz0ELyCCvrOacpqdg6Y2rqc+oYmZpaBtrn2ksCcALaM7tIBOZUkUo1Kza2kSo019V2kIShBQA)).